mod/wikirate/spec/lib/javascript/script_wikirate_common.test.coffee

```diff
@@ -0,0 +1,8 @@
+
+require('../../../lib/javascript/script_wikirate_common.js.coffee')
+
+test 'isString returns true for string', =>
+  expect($.wikirate.isString("ab")).toBeTruthy()
```

Comment:
I asked in a different PR (from Daniel) whether we wanted a convention for boolean functions in JS, since we can't use question marks.  I'm ok with `isX` and `hasX`.  are you?

Reply:
I think we should start shifting to camel in JS.  I kind of like that it helps signal a difference between JS and ruby.

Reply:
"start shifting" to camel? In my perception we have always used camel in JS and only a few snake variables sneaked in because we don't have a hound for enforcing that rule.

Reply:
fair enough. We just have to get over the camel's hump.